Improve challenge wording for `README.md` (#2)
freeCodeCamp/boilerplate-SHA-1-password-cracker: README.md

The function should hash each password from `top-10000-passwords.txt` and compare it to the hash passed into the function.

> The function should take an optional second argument named `use_salts`. If set to true, each salt string from the file `known-salts.txt` should check whether the salt is prepended **OR** appended to each password from `top-10000-passwords.txt` before hashing, and then comparing it to the hash passed into the function.

Reviewer: ```suggestion
The function should take an optional second argument named `use_salts`. If set to true, each salt string from the file `known-salts.txt` should separately be appended AND separately prepended to each password from `top-10000-passwords.txt` before hashing, and before comparing it to the hash passed into the function.
```
Perhaps something like this would be better, what do you think?

Author: I think the two additions of `separately` confuses the matter, and doesn't read particularly well.
The key problem here is the use of `AND`, because it suggests that you need to preppend _and_ append for every instance of `password`; which is the wrong idea. Using `OR` here instead clearly defines that you should preppend _or_ append, and therefore avoids the confusion.

Another improvement could be made in the examples made further down, by showing for the salted hashes what salts were used, and whether they were preppended or appended to the password. Happy to add this suggested change in their as well.

Reviewer: The more I'm looking at it, the more I feel relying on the `AND` or `OR`, especially capitalized for emphasis, might be part of the issue. What about going into direction where it's explicitly said each salt needs to be used twice for each password?

```suggestion
The function should take an optional second argument named `use_salts`. If set to true, each salt string from the file `known-salts.txt` should be used twice for each password from `top-10000-passwords.txt`, once with salt appended to the password and once with salt prepended, before hashing, and before comparing it to the hash passed into the function.
```

Adding to examples which salt was used and where also could help.
